2021-04-06 10:46:06,757 INFO    MainThread:29107 [wandb_init.py:_log_setup():292] Logging user logs to /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210406_104606-2uvumlob/logs/debug.log
2021-04-06 10:46:06,757 INFO    MainThread:29107 [wandb_init.py:_log_setup():293] Logging internal logs to /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210406_104606-2uvumlob/logs/debug-internal.log
2021-04-06 10:46:06,757 INFO    MainThread:29107 [wandb_setup.py:_flush():68] setting env: {}
2021-04-06 10:46:06,757 INFO    MainThread:29107 [wandb_setup.py:_flush():68] setting user settings: {}
2021-04-06 10:46:06,757 INFO    MainThread:29107 [wandb_setup.py:_flush():68] multiprocessing start_methods=fork,spawn,forkserver
2021-04-06 10:46:08,913 INFO    MainThread:29107 [wandb_run.py:_console_start():1265] atexit reg
2021-04-06 10:46:08,915 INFO    MainThread:29107 [wandb_run.py:_redirect():1135] redirect: SettingsConsole.REDIRECT
2021-04-06 10:46:08,917 INFO    MainThread:29107 [wandb_run.py:_redirect():1138] Redirecting console.
2021-04-06 10:46:08,917 INFO    MainThread:29107 [redirect.py:install():196] install start
2021-04-06 10:46:08,918 INFO    MainThread:29107 [redirect.py:install():211] install stop
2021-04-06 10:46:08,918 INFO    MainThread:29107 [redirect.py:install():196] install start
2021-04-06 10:46:08,918 INFO    MainThread:29107 [redirect.py:install():211] install stop
2021-04-06 10:46:08,918 INFO    MainThread:29107 [wandb_run.py:_redirect():1182] Redirects installed.
2021-04-06 10:46:13,512 INFO    MainThread:29107 [wandb_run.py:_atexit_cleanup():1238] got exitcode: 255
2021-04-06 10:46:13,514 INFO    MainThread:29107 [wandb_run.py:_restore():1210] restore
2021-04-06 10:46:13,514 INFO    MainThread:29107 [redirect.py:uninstall():215] uninstall start
2021-04-06 10:46:13,514 ERROR   stderr    :29107 [redirect.py:_pipe_relay():129] problem in pipe relay
Traceback (most recent call last):
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/lib/redirect.py", line 127, in _pipe_relay
    cb(name, data)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 634, in _console_callback
    self._backend.interface.publish_output(name, data)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/interface/interface.py", line 146, in publish_output
    self._publish_output(o)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/interface/interface.py", line 151, in _publish_output
    self._publish(rec)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/interface/interface.py", line 429, in _publish
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown
2021-04-06 10:46:13,515 INFO    MainThread:29107 [redirect.py:_stop():264] _stop: stdout
2021-04-06 10:46:13,517 INFO    stdout    :29107 [redirect.py:_pipe_relay():119] relay done saw last write: stdout
2021-04-06 10:46:13,517 INFO    MainThread:29107 [redirect.py:_stop():270] _stop closed: stdout
2021-04-06 10:46:13,517 INFO    stdout    :29107 [redirect.py:_pipe_relay():135] relay done done: stdout
2021-04-06 10:46:13,517 INFO    MainThread:29107 [redirect.py:_stop():276] _stop joined: stdout
2021-04-06 10:46:13,517 INFO    MainThread:29107 [redirect.py:_stop():278] _stop rd closed: stdout
2021-04-06 10:46:13,517 INFO    MainThread:29107 [redirect.py:uninstall():219] uninstall done
2021-04-06 10:46:13,517 INFO    MainThread:29107 [redirect.py:uninstall():215] uninstall start
2021-04-06 10:46:13,518 INFO    MainThread:29107 [redirect.py:_stop():264] _stop: stderr
2021-04-06 10:46:13,518 INFO    MainThread:29107 [redirect.py:_stop():270] _stop closed: stderr
2021-04-06 10:46:13,518 INFO    stderr    :29107 [redirect.py:_pipe_relay():119] relay done saw last write: stderr
2021-04-06 10:46:13,518 INFO    stderr    :29107 [redirect.py:_pipe_relay():135] relay done done: stderr
2021-04-06 10:46:13,518 INFO    MainThread:29107 [redirect.py:_stop():276] _stop joined: stderr
2021-04-06 10:46:13,518 INFO    MainThread:29107 [redirect.py:_stop():278] _stop rd closed: stderr
2021-04-06 10:46:13,518 INFO    MainThread:29107 [redirect.py:uninstall():219] uninstall done
2021-04-06 10:46:13,519 INFO    MainThread:29107 [wandb_run.py:_restore():1210] restore
2021-04-06 10:46:13,589 INFO    MainThread:29107 [internal.py:handle_exit():137] Internal process exited
